src/main/webapp/politicianPage.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="stylesheets/feedStyle.css">
```

Comment:
Will need a new css page for this html

Reply:
is this blocking?

Reply:
nah cause I'm making the UI for politician page so will just make a new css file